2024Robot_Team3484/src/main/include/subsystems/IntakeSubsystem.h

```diff
@@ -0,0 +1,55 @@
+#ifndef INTAKE_H
+#define INTAKE_H
+
+
+#include <frc2/command/SubsystemBase.h>
+#include <rev/CANSparkMax.h>
+//#include <ctre/Phoenix.h>
+#include <frc/DigitalInput.h>
+#include <frc/trajectory/TrapezoidProfile.h>
+
+#include <units/angle.h>
+
+#include <Constants.h>
+
+class IntakeSubsystem : public frc2::SubsystemBase {
+    public:
+        IntakeSubsystem(
+            int pivot_motor_can_id, 
+            int drive_motor_can_id, 
+            int piece_sensor_di_ch,
+            int arm_sensor_di_ch,
+
+            SC::SC_PIDConstants PID_CONSTANTS,
```

Comment:
nit: s/PID_CONSTANTS/pid_contants. or something similar. Technically this masks the PID_CONSTANTS declared in `Constants.h`, but also is a bit misleading if you are referring to the constants variable it self, or the constructor parameter.